src/pharmpy/modeling/summarize_individuals.py

```diff
@@ -0,0 +1,139 @@
+from typing import Callable, Iterable, Union
+
+import numpy as np
+import pandas as pd
+
+from pharmpy.model import Model
+from pharmpy.modeling.ml import predict_influential_individuals, predict_outliers
+from pharmpy.tools.modelsearch.tool import ModelSearchResults
+
+
+def summarize_individuals(models: list[Model]) -> Union[pd.DataFrame, None]:
+    """Creates a summary dataframe keyed by model-individual pairs for an input
+    list of models.
+
+    Parameters
+    ----------
+    models : list[Model]
+        Input models
+
+    Return
+    ------
+    pd.DataFrame | None
+        The summary as a dataframe
+
+    Examples
+    --------
+    >>> from pharmpy.modeling import *
+    >>> model = load_example_model("pheno")
+    >>> fit(model)
+    >>> results = run_tool(
+    >>>     'modelsearch',
+    >>>     model=model,
+    >>>     mfl='ABSORPTION(ZO);PERIPHERALS([1, 2])',
+    >>>     algorithm='reduced_stepwise'
+    >>> )
+    >>> summarize_individuals(results) # doctest: +SKIP
```

Comment:
This line need to be updated to reflect new API. Are we happy with a list of models as input? Should we check that there are no missing parents?

Reply:
I think a list is fine. If you want you could check for missing parents. In that case I case the diOFV couldn't be computed.